Source/FullScreenPokemon.ts

```diff
@@ -9196,6 +9196,35 @@ module FullScreenPokemon {
         }
 
         /**
+         * Pushes and saves the current state of a variable to a stack.
+         * 
+         * @param thing   The Thing, Area, Map, or Location saving its state of a variable.
+         * @param title   Name for the state being saved.
+         * @param information   The values of the variable to be saved.
+         */
+        addStateHistory(thing: IThing | IArea | IMap | ILocation, title: string, information: any): void {
+            var stateHistory: [any] = thing.state[title];
```

Comment:
This would be `any`[]. `[any]` is an array of length 1 with contents typed to `any`, whereas `any[]` is an array of any length with all contents typed to `any`.